client/src/modules/dashboard/Chapters/components/DeleteChapterButton.tsx

```diff
@@ -36,8 +36,14 @@ export const DeleteChapterButton = ({
     refetchQueries: [
       { query: CHAPTERS },
       { query: DASHBOARD_CHAPTERS },
-      { query: DASHBOARD_EVENTS, variables: { showCanceled: true } },
-      { query: DASHBOARD_EVENTS, variables: { showCanceled: false } },
+      {
+        query: DASHBOARD_EVENTS,
+        variables: { showCanceled: true, showRecent: true },
+      },
+      {
+        query: DASHBOARD_EVENTS,
+        variables: { showCanceled: false, showRecent: false },
+      },
```

Comment:
Event more I think it might be better to do filtering only in client... The `{ showCanceled: true, showRecent: false }` and `{ showCanceled: false, showRecent: true }` combinations won't be refetched without adding them explicitly. What starts to get a bit crazy.

Reply:
TD;LR, I agree we should sort this in the client.

I have created useFilter hook in this PR https://github.com/freeCodeCamp/chapter/pull/2301, that we can use to filter the Events page, it won't change much, will need to add `boolean` action in this PR, and it should result in the same functionality in the server.

When it's merged, I will finish this 👍